script/releases/v1.0.0-slashing/6-executeTimelock.s.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {MultisigCall, MultisigCallUtils, MultisigBuilder} from "zeus-templates/templates/MultisigBuilder.sol";
+import {SafeTx, SafeTxUtils} from "zeus-templates/utils/SafeTxUtils.sol";
+import {Queue} from "./2-queueUpgrade.s.sol";
+import {EigenLabsUpgrade} from "../EigenLabsUpgrade.s.sol";
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {RewardsCoordinator} from "src/contracts/core/RewardsCoordinator.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+contract Execute is Queue {
+    using MultisigCallUtils for MultisigCall[];
+    using SafeTxUtils for SafeTx;
+    using EigenLabsUpgrade for *;
+
+    /**
+     * @dev Overrides the previous _execute function to execute the queued transactions.
+     */
+    function _runAsMultisig() internal override {
+        bytes memory call = _getMultisigTransactionCalldata();
+        TimelockController timelock = TimelockController(payable(this._timelock()));
+        timelock.execute(
+            this._executorMultisig(),
+            0,
+            call,
+            0,
+            bytes32(0)
+        );
+    }
+
+    function testDeploy() override public {}
+
+    function testExecute() public {
+        // TODO: run 1-queueUpgrade + 2-queueUnpause, e.g via 
+    
+        // vm.startPrank(this._operationsMultisig());
+        // super._runAsMultisig();
```

Comment:
@wadealexc / @ypatil12 i need syntax to call a method from another contract without inheritance. since we're testing two upgrade scripts here, we need access to `_runAsMultisig()` and also `_ _getMultisigTransactionCalldata` from both of those scripts...

Reply:
or, we can do this as two separate phases, and just land one after the other with no changes

Reply:
wouldn't this inherit from the queue script? i thought that's how we set the templates up